spec/timezone.html

```diff
@@ -27,6 +29,29 @@ <h1>Time Zone Names</h1>
       Other implementations are encouraged to do the same.
     </p>
 
+    <emu-clause id="sec-time-zone-data-updates">
+      <h1>Handling Updates and Ensuring Consistency of Time Zone Data</h1>
+      <p>
+        The IANA Time Zone Database is frequently updated, averaging 10 annual releases in recent years.
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API is strongly encouraged to use the most recent version of the IANA Time Zone Database that was available when the implementation was released.
```

Comment:
```suggestion
        An ECMAScript implementation is strongly encouraged to use the most recent version of the IANA Time Zone Database that was available when the implementation was released.
```

Reply:
Cool. I pushed a new commit that includes this and your other suggestion, plus a few other minor edits.